### ISSUE TITLE
As an Mentor-OTP maintainer, I need to solve the deployment issues around Amarillo to avoid breaking the build pipeline indefinitely

### ISSUE THREAD

**Issue by clezag** [noi-techpark/odh-mentor-otp#212]
As mentioned in https://github.com/noi-techpark/odh-mentor-otp/pull/211, there is an issue with the OTP build process.

If the amarillo service ever fails for some reason, deploying a fix will also fail, since it's deployment relies on it's predecessor working.

We either need to have the OTP build use a separate local amarillo instance, or do a multi-stage deployment (e.g. first deploy amarillo, then build OTP) to have this work reliably.

**Comment by sseppi** [comment]
@clezag is this issue still actual?

**Comment by clezag** [comment]
No, since now the graph builds in parallel after deployment and not during ci/cd
